src/tests/performance/connections.rs

```diff
@@ -187,15 +203,15 @@ async fn p002_connections_load() {
         metrics::register_counter!(METRIC_REJECTED);
         metrics::register_counter!(METRIC_ERROR);
 
-        let mut synth_handles = Vec::with_capacity(synth_count as usize);
-        let mut synth_exits = Vec::with_capacity(synth_count as usize);
+        let mut synth_handles = Vec::with_capacity(synth_count.into());
+        let mut synth_exits = Vec::with_capacity(synth_count.into());
         let (handshake_tx, mut handshake_rx) =
             tokio::sync::mpsc::channel::<()>(synth_count as usize);
 
         let test_start = Instant::now();
 
         // start synthetic nodes
-        for _ in 0..synth_count {
+        for socket in synth_sockets.into_iter() {

```

Comment:
Why is `into_iter` needed?

Reply:
Ah, right, `for` loops use `into_iter` by default